TATB-48 [BUG] : Getting console error with empty page [feature/development]

PrabhatJTDeveloper opened the pull request.

PrabhatJTDeveloper committed:
- TATB 45
  - redesign the result page and for that change the html and css file
  - remove not used images and add new images.

PrabhatJTDeveloper committed:
- TATB 48
  - make component of page not found
  - add the component to the routing module and layout component and layout rounting

PrabhatJTDeveloper committed:
- TATB 48
  - changed the design of page and for that changed the html and css file.

PoojanPatelJT commented on src/app/component/quiz/quiz.component.ts:
```diff
@@ -72,24 +74,28 @@ export class Quizcomponent implements OnInit, OnDestroy {
+      this.timer = quizData?.timer;
+      this.positivePoints = quizData?.positivePoints;
+      this.negativePoints = quizData.negativePoints;
+  
```
please add proper undefined check

PrabhatJTDeveloper replied:
ok, i added proper undefined check.

PoojanPatelJT commented on src/app/component/quiz/quiz.component.ts:
```diff
@@ -72,24 +74,28 @@ export class Quizcomponent implements OnInit, OnDestroy {
+    if (!quizData) {
+       this.router.navigateByUrl('404Page');
+    }else{
+      this.question = [...quizData.questions];
```
give proper indentation

PrabhatJTDeveloper replied:
ok, i have done code indentation

PoojanPatelJT commented on src/app/component/result/result.component.scss:
```diff
@@ -1,59 +1,27 @@
-  font-weight: bold;
-  padding-top: 10px;
+.container{
+  .result-data{
```
here also give indentation

PrabhatJTDeveloper replied:
ok, i have done code indentation

PoojanPatelJT commented on src/app/component/quiz/quiz.component.ts:
```diff
@@ -58,6 +58,8 @@ export class Quizcomponent implements OnInit, OnDestroy {
     this.startCounter();
     this.getQuestionData();
+
+  
```
Why here multiple space added here

PrabhatJTDeveloper replied:
i removed the space and done code indentation

PrabhatJTDeveloper committed:
- TATB 48
  - Done code indentation in quiz component ts file and result component scss file

charmijtdev commented on src/app/authorization/login/login.component.html:
```diff
@@ -54,9 +54,10 @@
+              *ngIf="(loginFormControl['password']?.errors?.['pattern'])"
             >
-              Password should be at least 6 and at the max 12 characters long
+              Password should be atleast 8 characters long and should contain
```
please write in bullet points

PrabhatJTDeveloper replied:
ok, i changed it.

charmijtdev commented on src/app/component/quiz/quiz.component.ts:
```diff
@@ -11,15 +11,15 @@ import { interval, Subscription, ReplaySubject, takeUntil } from 'rxjs';
   selector: 'app-questions',
   templateUrl: './quiz.component.html',
-  styleUrls: ['./quiz.component.scss']
+  styleUrls: ['./quiz.component.scss'],
```
please remove comma from here

PrabhatJTDeveloper replied:
ok, i removed it.

charmijtdev commented on src/app/component/result/result.component.scss:
```diff
@@ -1,59 +1,23 @@
-.card-img-top {
-  max-height: 120px;
+.cardContainer {
+  max-height: 380px !important;
```
px ? I guess in large screen card would be small i guess did you checked it ?

PrabhatJTDeveloper replied:
As this is not require i removed it.

charmijtdev commented on src/app/component/result/result.component.scss:
```diff
@@ -1,59 +1,23 @@
-  padding: 15px;
-  margin-top: 80px;
+.img-content {
+  min-height: 140px;
```
please use scss type

PrabhatJTDeveloper replied:
ok , i changed it.

charmijtdev commented on src/app/app-routing.module.ts:
```diff
@@ -17,7 +17,7 @@ const routes: Routes = [
       ),
   },
-  { path: '**', redirectTo: 'login', pathMatch: 'full' },
+  { path: '**', component: PageNotFoundComponent,  pathMatch: 'full'}
```
can we simply redirect to 404Page ?

PrabhatJTDeveloper replied:
yes, i changed it.

PrabhatJTDeveloper committed:
- TATB 48
    - change the style of result page
    - change the path of 404page in app-routing

PrabhatJTDeveloper committed:
Merge branch 'feature/stagging-all-PR-changes' of https://github.com/Jyoti-Technosoft/technical-assistant into bug/TATB-48-bug-getting-console-error-with-empty-page

PrabhatJTDeveloper committed:
TATB-48  : Take pull of stagging all pr branch

PrabhatJTDeveloper committed:
- TATB 48
     - added the button on the 404Page for navigating to the dashboard.
     - update the path for the 404Page in layout-routing file.

PrabhatJTDeveloper committed:
- TATB 48
    - solve the conflicts.

PrabhatJTDeveloper committed:
Merge branch 'feature/stagging-all-PR-changes' of https://github.com/Jyoti-Technosoft/technical-assistant into bug/TATB-48-bug-getting-console-error-with-empty-page

PrabhatJTDeveloper committed:
TATB-48 : Fixed merge conflicts

PrabhatJTDeveloper committed:
- TATB 48
  - solved the conflicts
  - solved the error in quiz component

PrabhatJTDeveloper committed:
- solve the conflict

PrabhatJTDeveloper committed:
- TATB 48
   - change the error printing in login form
   - remove extra comma from quiz component ts file.

PrabhatJTDeveloper committed:
- solve the conflict

PrabhatJTDeveloper committed:
- TATB 48
  - solve the issue of 404Page showing in result and all result page.
